source/renderer/app/components/status/DaedalusDiagnostics.scss

```diff
@@ -230,7 +224,11 @@
 =            SPECIFIC STYLES            =
 =======================================*/
 
-td.platformVersion {
+.layoutHeader.stateDirectoryPath {
+  width: 244px;
```

Comment:
@DeeJayElly can we avoid hardcoding width here?

Reply:
Well the solution with flexbox works but only if we have a width set on the left element inside the left/right columns on the Diagnostic screen and we need to have it here.